lib/active_model/serializer/adapter.rb

```diff
@@ -50,6 +50,20 @@ def include_meta(json)
         json[meta_key] = meta if meta && root
         json
       end
+
+      private
+
+      def cached_object
+        klass = serializer.class
+        if klass._cache
+          _cache_key = klass._cache_key || serializer.object.class.name.downcase
+          klass._cache.fetch("#{_cache_key}-#{serializer.object.id}", klass._cache_options) do
```

Comment:
I think I wasn't clear in my previous comment (https://github.com/rails-api/active_model_serializers/pull/693/files#diff-04c6e90faac2675aa89e2176d2eec7d8R252). Basically, I believe we should use `model.cache_key` here when key is not used (docs in http://api.rubyonrails.org/classes/ActiveRecord/Integration.html). If you don't specify a key, it'd use `object.cache_key`:

```
Person.find(5).cache_key  # => "people/5-20071224150000"
```

The problem with using only an id here is that if we update the model, it'll reuse an old cache because `update_at` is disregarded. With `cache_key`, it'll regard `update_at`. With a custom `key`, I believe `updated_at` should also be included in this identifier.

What do you think?